### ISSUE TITLE
Inaccurate "accuracy" when testing uploaded model.

### ISSUE THREAD

**Issue by SwanseaLeo** [kakaobrain/fast-autoaugment#54]
I run testing code by using your provided model. 
Cifar10
[Wide-ResNet-28-10 | 3.9 | 3.1 | 2.6 | 2.7 / 2.7 | Download] 
Cifar100
[Wide-ResNet-28-10 | 18.8 | 18.4 | 17.1 | 17.3 / 17.3 | Download]
But I can't get the paper's results? Is something wrong ? 
Looking forward to your reply, thank you~
The results of cifar10 are below:
[2020-11-12 06:10:48,729] [Fast AutoAugment] [WARNING] tag not provided, no tensorboard log.
[2020-11-12 06:10:48,730] [Fast AutoAugment] [INFO] ./FAA_Paper_models/cifar10_wresnet28x10_top1.pth file found. loading...
[2020-11-12 06:10:48,934] [Fast AutoAugment] [INFO] checkpoint epoch@10
[2020-11-12 06:10:48,941] [Fast AutoAugment] [INFO] optimizer.load_state_dict+
[2020-11-12 06:10:48,950] [Fast AutoAugment] [INFO] evaluation only+
[2020-11-12 06:11:57,150] [Fast AutoAugment] [INFO] done.
[2020-11-12 06:11:57,151] [Fast AutoAugment] [INFO] model: {'type': 'wresnet28_10'}
[2020-11-12 06:11:57,151] [Fast AutoAugment] [INFO] augmentation: fa_reduced_cifar10
[2020-11-12 06:11:57,151] [Fast AutoAugment] [INFO] 
{
    "loss_train": NaN,
    "loss_valid": 0.0,
    "loss_test": NaN,
    "top1_train": 0.09475160256410256,
    "top1_valid": 0.0,
    "top1_test": 0.0909,
    "top5_train": 0.4834735576923077,
    "top5_valid": 0.0,
    "top5_test": 0.4729,
    "epoch": 0
}
[2020-11-12 06:11:57,152] [Fast AutoAugment] [INFO] elapsed time: 0.021 Hours
[2020-11-12 06:11:57,152] [Fast AutoAugment] [INFO] top1 error in testset: 0.9091
[2020-11-12 06:11:57,152] [Fast AutoAugment] [INFO] ./FAA_Paper_models/cifar10_wresnet28x10_top1.pth
